packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,69 @@
+import { BaseInput } from '../BaseInput';
+import type { BaseInputProps } from '../BaseInput';
+import { HiddenInput } from './HiddenInput';
+import type { IconComponent } from '~components/Icons';
+import { ChevronDownIcon, ChevronUpIcon } from '~components/Icons';
+import Box from '~components/Box';
+type SelectInputProps = Pick<
+  BaseInputProps,
+  | 'label'
+  | 'labelPosition'
+  | 'necessityIndicator'
+  | 'validationState'
+  | 'helpText'
+  | 'errorText'
+  | 'successText'
+  | 'placeholder'
+  | 'name'
+  | 'isDisabled'
+  | 'isRequired'
+  | 'prefix'
+  | 'suffix'
+  | 'autoFocus'
+> & {
+  icon?: IconComponent;
+};
+
+const SelectInput = (props: SelectInputProps): JSX.Element => {
+  /**
+   * @TODO handle expanded state with Dropdown
+   */
+  const isPopupExpanded = false;
+
+  const { icon, ...baseInputProps } = props;
+
+  return (
+    <Box position="relative">
+      {/* @TODO Use this for form submissions */}
+      <HiddenInput
+        required={props.isRequired}
+        value=""
+        // Accessibility is covered in the select input itself so we hide this field from a11y tree
+        aria-hidden={true}
```

Comment:
It'll be better to just directly add this on the HiddenInput.web.ts file instead of here, since this file will execute for both native & web.

Reply:
I removed that file. Using `VisuallyHidden` now with if else check

Reply:
fixed it in new commit